packages/api/tests/endpoints/providers/update-provider.js

```diff
@@ -100,21 +100,15 @@ test.serial('PUT updates an existing provider and returns it in listing', async
     .set('Authorization', `Bearer ${jwtAuthToken}`)
     .expect(200);
 
-  t.plan(2);
-  const stub = sinon.stub(Search.prototype, 'query').resolves({
-    results: [updatedProvider]
-  });
-
+  t.plan(1);
```

Comment:
If the test isn't stubbing anymore, I don't think it still needs to be `test.serial`?